sha1-utils.js

```diff
@@ -2,8 +2,23 @@
 
 const crypto = require("crypto");
 
-function getSha1(email) {
-  return crypto.createHash("sha1").update(email).digest("hex");
+// Email addresses must be lowercased before hashing
+// Hashes sent to HIBP should be uppercased
+function getSha1ForHIBP (email) {
+  const formattedEmail = email.toLowerCase();
+  const emailHash = crypto.createHash("sha1").update(formattedEmail).digest("hex");
```

Comment:
Silly Q: Thoughts on replacing some of this w/ `return getLowerCaseSha1(email).toUpperCase()`? and keeping it somewhat DRYer?